llvm/lib/Passes/PassRegistry.def

```diff
@@ -449,6 +449,7 @@ FUNCTION_PASS("expand-large-div-rem", ExpandLargeDivRemPass(TM));
 FUNCTION_PASS("expand-large-fp-convert", ExpandLargeFpConvertPass(TM));
 FUNCTION_PASS("dwarfehprepare", DwarfEHPreparePass(TM));
 FUNCTION_PASS("callbrprepare", CallBrPreparePass());
+FUNCTION_PASS("safe-stack", SafeStackPass(TM));
```

Comment:
Very minor nit: At some point it looks like these passes were listed alphabetically (all the non-alphabetized ones at the bottom have been added pretty recently). Do you mind moving this one up to prevent this one from being out of order?

Reply:
Thanks!